Projects/Server/GameCounter.h

```diff
@@ -0,0 +1,43 @@
+#ifndef GAMECOUNTER_H
+#define GAMECOUNTER_H
+
+#include "State.h"
+#include "GameHandler.h"
+
+#include <map>
+
+
+
+class GameCounter {
+public:
+	GameCounter(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& gameSample);
+	
+	friend std::ostream& operator<<(std::ostream& os, const GameCounter& gameCounter);
+
+	void play();
+
+	size_t getP0Win() const;
+	size_t getP1Win() const;
+	size_t getDraw() const;
+	size_t getGamesPlayed() const;
+	size_t getGameSample() const;
```

Comment:
What's the purpose of the below 4 getter? I don't think there is a valid reason why anybody should be able to get these from GameCounter. Please remove them, unless there is a good reason to implement it.

Reply:
They are not needed right now. But, what I thought was that if there would be a case to create a new class which inherits from this one to implement a new feature we could use these. However, I suppose these are unnecessary at this point. I will remove them shortly.

Reply:
These four functions have been removed.